examples/realTimeVideo-config/src/main.ts

```diff
@@ -0,0 +1,94 @@
+import { Authentication, Fleet, KeyValue } from "@formant/data-sdk";
+import "@formant/ui-sdk-joystick";
+import "@formant/ui-sdk-realtime-player";
+import { RealtimePlayer } from "@formant/ui-sdk-realtime-player";
+import "./style.css";
+
+const loadVideo = async (defaultCamera: number) => {
+  (el("formant-realtime-player") as RealtimePlayer).drawer.start();
+  let indicator = document.getElementById("loading-indicator");
+  try {
+    if (await Authentication.waitTilAuthenticated()) {
+      const device = await Fleet.getCurrentDevice();
+
+      await device.startRealtimeConnection();
+      device.addRealtimeListener((_peerId: any, message: any) => {
+        (el("formant-realtime-player") as RealtimePlayer).drawVideoFrame(
+          message.payload.h264VideoFrame
+        );
+      });
+      let videoStreams = await device.getRealtimeVideoStreams();
+      let settingsIcon = document.getElementById("settings");
+      let settingsBlock = document.getElementById("video-settings");
+
+      settingsIcon!.style.display = "flex";
+      settingsIcon?.addEventListener("click", () => {
+        settingsBlock!.style.display = "flex";
+      });
+      renderCameraOptions(videoStreams);
+
+      device.startListeningToRealtimeVideo(videoStreams[defaultCamera]);
+      setTimeout(() => {
+        indicator!.style.display = "none";
+      }, 3000);
+      el("formant-realtime-player").style.display = "block";
+    }
+  } catch (e) {
+    log((e as Error).message);
+  }
+};
+
+function saveDefaultCamera() {
+  const videoSource = document.querySelectorAll("p.camera-option");
+  videoSource.forEach((video) => {
+    video.addEventListener("click", (event) => {
+      const camera = (event.target as HTMLParagraphElement).id;
+      KeyValue.set("defaultVideoStream", camera.at(-1)!);
+      let settingsBlock = document.getElementById("video-settings");
+      settingsBlock!.style.display = "none";
+      location.reload();
+    });
+  });
+}
+
+function renderCameraOptions(videoStreams: any[]) {
+  videoStreams.forEach((_, idx) => {
+    $(".camera-options").append(
+      `<p id=${`video-${idx}`} class="camera-option">${_.name}</p>`
+    );
+  });
+}
+
+function log(msg: string) {
+  el("#log").innerHTML = msg + "<br>" + el("#log").innerHTML;
+}
```

Comment:
An easy way to do this without having to muck with `innerHTML`, is to use `append()`, or more specifically [`prepend()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/prepend), on `Element`. This can be rewritten as:

```ts
const log = (msg: string) => el("#log").prepend(msg, document.createElement("br"));
```